internal/serve/httphandler/assets_handler_test.go

```diff
@@ -1456,7 +1456,7 @@ func Test_AssetHandler_submitChangeTrustTransaction_makeSurePreconditionsAreSetA
 			timeSinceCreation := time.Since(creationTime)
 
 			expectedAdjustedMaxTime := expectedMaxTime.Add(timeSinceCreation)
-			require.WithinDuration(t, expectedAdjustedMaxTime, actualMaxTime, 5*time.Second)
+			require.WithinDuration(t, expectedAdjustedMaxTime, actualMaxTime, 10*time.Second)
```

Comment:
Unrelated change. This test fails randomly for me with durations close to 5 (5.5-6).